internal/server/spanner/statements.go

```diff
@@ -291,6 +282,18 @@ var statements = struct {
 			Observation,
 			UNNEST(observations) as obs
 	`,
+	allObs: `
+		observations
+	`,
+	latestObs: `
+		[observations[ARRAY_LENGTH(observations)-1]] AS observations
+	`,
+	dateObs: `
+		[obs] AS observations
```

Comment:
(sorry for confusion, looks like arrays are needed after all, since some tests were failing)